Src/Sarif.PatternMatcher/SearchSkimmer.cs

```diff
@@ -226,6 +226,8 @@ public override void Analyze(AnalyzeContext context)
                     // The validator determined the match is a false positive.
                     // i.e., it is not the kind of artifact we're looking for.
                     // We should suspend processing and move to the next match.
+
+                    level = FailureLevel.Note;
```

Comment:
>Note [](start = 41, length = 4)

This isn't right, no match is not a failure. This should be 'None' if you want an explicit value.